src/rgw/driver/rados/rgw_rados.cc

```diff
@@ -4832,15 +4832,7 @@ int RGWRados::copy_obj(RGWObjectCtx& obj_ctx,
   RGWRados::Object dest_op_target(this, dest_bucket_info, obj_ctx, dest_obj);
   RGWRados::Object::Write write_op(&dest_op_target);
 
-  string tag;
-
-  if (ptag) {
```

Comment:
if you arew not using `ptag` anymore,  why not moving it from the function prototype?